auto.config.ts

```diff
@@ -0,0 +1,18 @@
+import { AutoRc } from "auto";
+
+
+/** Auto configuration */
+export default function rc(): AutoRc {
+    return {
+        plugins: [
+            "npm",
+            "conventional-commits",
+            "first-time-contributor",
+            "released"
+        ],
+        // Follow the 2 branch deployment scheme
+        // https://intuit.github.io/auto/docs/generated/shipit#next-branch-default
+        baseBranch: "stable",        // latest "official" release
+        prereleaseBranches: ["main"] // latest changes (subject to breaking). next, alpha, beta, and other multi-feature test branches can all be added here.
```

Comment:
Haven't edited this config yet, but plan to make switch this to `next` after confirming it makes sense w/ @domoritz to rotate this to the default branch

Reply:
Rotating the default branch makes sense to me.

Reply:
Updated to `next`: https://github.com/vega/ts-json-schema-generator/pull/856/commits/492820bcf73b271a1a4c233031be8a41b84e6717

I don't have admin permissions on this repo, so I'll need your help to switch the default branch to next. I just synced next to the HEAD of `master` (`8439cfbb45918c11ac5df65e9a516a07763478a9`) so we should be ready to go.